FPM.ftd

```diff
@@ -123,6 +123,9 @@ email: arpita@fifthtry.com
     url: processors/document-readers/
   - `$processor$: document-writers`
     url: processors/document-writers/
+  - `$processor$: document-filename`
+    url: processors/document-filename/
```

Comment:
In sitemap all urls should start and end with slash. We like ids like /foo/, not foo/ nor /foo.

Reply:
Existing entries are wrong as well. Fix all of them. Or at least use the right format now on.